### ISSUE TITLE
Use of Bikeshed for non-W3C organization

### ISSUE THREAD

**Issue by cconcolato** [tabatkins/bikeshed#2026]
The [Alliance for Open Media (AOM) ](http://aomedia.org/) uses Bikeshed for some of its specifications, and it's working great, thank you. See some examples: [AV1-ISOBMFF](https://aomediacodec.github.io/av1-isobmff/), [AVIF](https://aomediacodec.github.io/av1-avif/). 

However, the only available statuses are in the "Anyone" category as indicated in the [documentation](https://tabatkins.github.io/bikeshed/#metadata). They don't match the possible statuses of the AOM's documents, which are derived from the [Joint Development Foundation](https://www.jointdevelopment.org/) process. This means that when the spec gets final, I have to edit the generated HTML to remove the "Living Standard" status to something more formal (version number). 

Would it be possible to add an AOM or JDF group, with the following status: "Pre-Draft", "Draft Deliverable", "Final Deliverable", "Published Specification"?

I would be happy to know if other hooks exist for external organizations to customize/remove W3C-specific behavior (e.g. use of `'` for CSS properties).

**Comment by tabatkins** [comment]
> Would it be possible to add an AOM or JDF group, with the following status: "Pre-Draft", "Draft Deliverable", "Final Deliverable", "Published Specification"?

Absolutely. What short names do you want for those statuses? And what's your group name, so we can add some boilerplate for you? (You'll need to give me the boilerplate files.)

> I would be happy to know if other hooks exist for external organizations to customize/remove W3C-specific behavior (e.g. use of ' for CSS properties).

You can turn the various shorthands on or off with [`Markup Shorthands`](https://tabatkins.github.io/bikeshed/#metadata-markup-shorthands)

**Comment by cconcolato** [comment]
> Absolutely. What short names do you want for those statuses? And what's your group name, so we can add some boilerplate for you? (You'll need to give me the boilerplate files.)

I discussed it within AOM and we'd like the following short names and statutes:
* PD - "Pre-Draft"
* WGD - "AOM Working Group Draft"
* WGA - "AOM Working Group Approved Draft" 
* FD - "AOM Final Deliverable" 
The group name should AOM. 
For the boilerplate, do you have an example?

> You can turn the various shorthands on or off with Markup Shorthands

Nice. Thanks. I will try that.

**Comment by cconcolato** [comment]
@tabatkins do you have enough information to start?

**Comment by cconcolato** [comment]
@tabatkins how can we progress on the use of Bikeshed for AOM?
